src/database/config.go

```diff
@@ -0,0 +1,19 @@
+package database
+
+import(
+	"github.com/kamva/mgm/v3"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+)
+
+const MONGODB_URI = "mongodb://127.0.0.1:27017/?directConnection=true&serverSelectionTimeoutMS=2000"
+const DB_NAME = "comiditapp_db"
```

Comment:
¿Esto no debería de estar en un .env?

Reply:
Realmente no, porque si añadieramos esos cambios quedaría tal que así, y los datos que estamos mostrando ahora mismo serían para desarrollo en local
```suggestion
const MONGODB_URI = env.MONGODB_URI || "mongodb://127.0.0.1:27017/?directConnection=true&serverSelectionTimeoutMS=2000"
const DB_NAME = env.DB_NAME || "comiditapp_db"
```